pandas/tests/io/excel/test_odf.py

```diff
@@ -36,3 +36,16 @@ def test_read_writer_table():
     result = pd.read_excel("writertable.odt", sheet_name="Table1", index_col=0)
 
     tm.assert_frame_equal(result, expected)
+
+
+def test_read_newlines_between_xml_elements_table():
+    # Also test reading table from an text OpenDocument file
```

Comment:
Please add gh reference with GH#number

Reply:
You don‘t need the comment, the reference is enough

Reply:
Done! But thanks! Good to know for my next PRs!